mass_merge/models/base_merge_model_lines.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2020 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class BaseMergeModelLine(models.Model):
+
+    _name = 'base.merge.model.line'
+
+    merge_model_id = fields.Many2one(
```

Comment:
A many2one will needs it's corresponding model. Otherwise it will throw `_unknown` error

Reply:
Model for Many2one Added